packages/terra-date-time-picker/tests/nightwatch/date-time-picker-spec.js

```diff
@@ -0,0 +1,271 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+/* eslint-disable no-unused-expressions */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const screenshot = require('terra-toolkit').screenshot;
```

Comment:
One thing to note about the switched to ChromeDriver. When using click, it will click the center of the element specified. Not sure if/how that will affect selecting these imputs.